jib-core/src/test/java/com/google/cloud/tools/jib/docker/DockerClientTest.java

```diff
@@ -56,8 +56,13 @@ public void setUp() throws IOException {
   public void testIsDockerInstalled_pass() throws URISyntaxException, IOException {
     Path executableFile = temporaryFolder.newFolder().toPath().resolve("executable");
     Files.copy(Paths.get(Resources.getResource("executable").toURI()), executableFile);
-    // 755
-    Files.setPosixFilePermissions(executableFile, PosixFilePermissions.fromString("rwxr-xr-x"));
+    try {
+      // 755
+      Files.setPosixFilePermissions(executableFile, PosixFilePermissions.fromString("rwxr-xr-x"));
+
+    } catch (UnsupportedOperationException ex) {
```

Comment:
We probably want to avoid catching a runtime exception.

Can you try to get the AttributeView and check for nullness instead?

Reply:
I opted to remove this executable-running and just use a mock instead since having a separate executable file and logic for Windows seemed like overkill for a single test.